install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -538,15 +538,22 @@ data:
   enable-auto-protect-node-port-range: {{ .Values.nodePort.autoProtectPortRange | quote }}
 {{- end }}
 {{- if hasKey .Values "loadBalancer" }}
+{{- if .Values.loadBalancer.standalone }}
```

Comment:
Should we have this value added as a comment into `values.yml`?

Reply:
Thanks, added!